src/components/ScenarioParameters/FileManagementUtils.js

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { UPLOAD_FILE_STATUS_KEY } from '@cosmotech/ui';
+import { ORGANIZATION_ID, WORKSPACE_ID } from '../../config/AppInstance';
+import DatasetService from '../../services/dataset/DatasetService';
+import WorkspaceService from '../../services/workspace/WorkspaceService';
+import { AppInsights } from '../../services/AppInsights';
+import { DatasetsUtils, ScenarioParametersUtils } from '../../utils';
+
+export const STORAGE_ROOT_DIR_PLACEHOLDER = '%WORKSPACE_FILE%/';
+const DATASET_ID_VARTYPE = '%DATASETID%';
```

Comment:
`%DATASETID%` is present in several files, it could be interesting to create a const in a config file under services/config and use this const